docs/docs/intro.md

```
@@ -0,0 +1,47 @@
+## Start your site
+
+Run the development server:
+
+```bash
+cd my-website
+npm run start
+```
+
+The `cd` command changes the directory you're working with. In order to work with your newly created Docusaurus site, you'll need to navigate the terminal there.
+
+The `npm run start` command builds your website locally and serves it through a development server, ready for you to view at http://localhost:3000/.
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)